packages/test/end-to-end-tests/src/test/deRehydrateContainerTests.spec.ts

```diff
@@ -55,7 +55,7 @@ describe(`Dehydrate Rehydrate Container Test`, () => {
         const container = await loader.createDetachedContainer(codeDetails);
         // Get the root dataStore from the detached container.
         const response = await container.request({ url: "/" });
-        const defaultDataStore = response.value;
+        const defaultDataStore = response.value as TestFluidObject;
```

Comment:
Any chance you can take a look how to change this code to using requestFluidObject?
Basically, I want our code to never ever need to check response and use the layer above it that abstracts away (and eventually hopefully remove things like checking for status-codes, mimeType, and code like this that does not do it properly :)

It assumes IFluidRouter which is not implemented by IContainer. We should either make it implement this interface, or reduce that requirement on requestFluidObject, i.e. maybe only require an interface that has request() method on it and that's it.

Thanks!

Reply:
BTW, that might be separate PR :) It would be nice to go through all of our code and stop dealing with responces - fee free to open an issue

Reply:
I can but I will keep that separate from this PR.